Resources/Prototypes/Entities/Mobs/Player/human.yml

```diff
@@ -26,6 +26,7 @@
   - type: Alerts
   - type: Actions
   - type: Eye
+  - type: ContentEye
```

Comment:
I think you can just add the component for `Resources/Prototypes/Entities/Mobs/Species/base.yml` instead of copy pasting it for all the species

Reply:
fixed